lang/java/avro/src/test/java/org/apache/avro/TestLogicalType.java

```diff
@@ -59,8 +59,7 @@ public void testDecimalWithNonByteArrayTypes() {
         Schema.createEnum("Enum", null, null, Arrays.asList("a", "b")),
         Schema.createUnion(Arrays.asList(Schema.create(Schema.Type.BYTES), Schema.createFixed("fixed", null, null, 4))),
         Schema.create(Schema.Type.BOOLEAN), Schema.create(Schema.Type.INT), Schema.create(Schema.Type.LONG),
-        Schema.create(Schema.Type.FLOAT), Schema.create(Schema.Type.DOUBLE), Schema.create(Schema.Type.NULL),
-        Schema.create(Schema.Type.STRING) };
+        Schema.create(Schema.Type.FLOAT), Schema.create(Schema.Type.DOUBLE), Schema.create(Schema.Type.NULL) };
```

Comment:
I think we should rename the test from `testDecimalWithNonByteArrayTypes` to: `testDecimalWithInvalidTypes`

Reply:
done